test/unit/config/enhance-octokit.test.js

```diff
@@ -1,25 +1,70 @@
 const Octokit = require('@octokit/rest')
 const nock = require('nock')
+const LogDouble = require('../../setup/log-double')
 
 const enhanceOctokit = require('../../../lib/config/enhance-octokit')
 
 describe(enhanceOctokit, () => {
   describe('request metrics', () => {
-    it('sends reqest timing', async () => {
-      nock('https://api.github.com').get(/.+/).reply(200, [])
+    let log
+    let octokit
 
-      const log = { debug: () => {} }
-      const octokit = Octokit()
+    beforeEach(() => {
+      log = new LogDouble()
+      octokit = Octokit()
+    })
+
+    describe('when successful', () => {
+      beforeEach(() => {
+        nock('https://api.github.com').get(/.+/).reply(200, [])
+        enhanceOctokit(octokit, log)
+      })
 
-      enhanceOctokit(octokit, log)
+      it('sends reqest timing', async () => {
+        await expect(async () => {
+          await octokit.activity.listPublicEvents()
+        }).toHaveSentMetrics({
+          name: 'jira-integration.github-request',
+          type: 'h',
+          value: (value) => value > 0 && value < 20, // Value changes depending on how long nock takes
```

Comment:
> long nock takes

mock?

Reply:
https://github.com/nock/nock

It's an HTTP mocking library for Node, so `Node.js + Mock`? 🤷‍♂ 

One of those typo-looking names. 😄